src/IntrType.f90

```diff
@@ -139,6 +142,41 @@ MODULE IntrType
 INTEGER,PARAMETER :: SRK=SSK
 #endif
 
+!> @brief Mixed-precision REAL kind
+!>
+!> Optionally allow single precision for the variables of large
+!> memory footprint. SGK, SCK and SFK are defined for different
+!> variables of interest and may be changed separately for testing.
+!> The scaling factors (FSCALE#) are only used in unit tests to
```

Comment:
They are used to preserve the checks in unit tests for double precision (factors = 1) and increase the tolerance for mixed precision. We can finally remove them when we decide to go with mixed precision, but they are helpful in the transition period. They are defined up to 10e9 since many of the tolerances used in unit tests are rather arbitrary (say 10e-13, 10e-10, 10e-6, etc.). I will move these definitions to MPACTUtils too.

Reply:
So am I understanding you correctly that you intend to only have 1 FSCALE value that will be used for all tests, or some places in the test will use FSCALE3, another test will use FSCALE4, etc?  I do feel like this name as a global in everything we do is a little troubling because its very generically named.  What I don't want to see is someone starting to use FSCALE6 somewhat arbitrarily in the code when they need to multiply by 1e-6 (and since we eventually will run mixedprecision by default, it will work 99% of the time... until someone turns off mixed precision).  I'd also recommend renaming this to be more clear: MIXEDPRECISION_MULTIPLIER6 (as an example.  But I'd almost prefer these to be defined locally in the test itself.

Reply:
The tolerances of many unit tests are pretty arbitrary so I had to use different scale factors for different tests unless I used a large one (FSCALE9) then all tests will pass for sure. I can change the name and probably define them closer to unit tests if possible (don't really want to repeat definitions of them in each unit test since there are really a lot of use of them.